docs/data-management/intro.md

```diff
@@ -119,4 +124,4 @@ The above graph can be detailed into two main data workflows:
 1. **MRI data**.
     Stored on the CHUV PACS, the raw MRI data should first be locally [converted to BIDS](post-session.md/#convert-imaging-data-to-bids-with-heudiconv). Then both the raw and BIDS compliant data are to be synchronized with the [*Datalad* dataset](preliminary.md/#adding-data-or-metadata). From there, the BIDS compliant data can be [quality controlled](./mriqc.md) and [pre-processed](../processing/preprocessing.md) using the corresponding packages (e.g. *MRIQC*, *fMRIPrep*, *dMRIPrep*) to allow computation of analysis-grade derivatives (e.g. [functional](../processing/functional-connectivity.md) or structural connectivity).
 2. **Physiological data**.
-    Physiological data stored in the *stimuli presentation laptop* ({{ secrets.hosts.psychopy | default("███") }}) and the *physio-recording Laptop* ({{ secrets.hosts.acqknowledge | default("███") }}) are synchronized with a cloud storage using *Dropbox*. The *AcqKnowledge* and ET physiological data are converted to BIDS using [*phys2bids*](physio-to-bids.ipynb) and *etf2bids* respectively. **Further processing steps such as denoising should be defined soon.**
\ No newline at end of file
+    Physiological data stored in the *stimuli presentation laptop* ({{ secrets.hosts.psychopy | default("███") }}) and the *physio-recording Laptop* ({{ secrets.hosts.acqknowledge | default("███") }}) are synchronized with a cloud storage using *Dropbox*. The *AcqKnowledge* and ET physiological data are converted to BIDS using [*phys2bids*](physio-to-bids.ipynb) and *etf2bids* respectively. The logs from *psychopy* are converted into BIDS `_events.tsv` files using the [*write_event_file.py*](./post-session.md/#generate-bids-events-files) script. **Further processing steps such as denoising should be defined soon.**
```

Comment:
Please use semantic lines (https://sembr.org/)
```suggestion
    Physiological data stored in the *stimuli presentation laptop* ({{ secrets.hosts.psychopy | default("███") }}) and the *physio-recording laptop* ({{ secrets.hosts.acqknowledge | default("███") }}) are synchronized with a cloud storage using *Dropbox*.
    The physiological signals (*AcqKnowledge*) and ET's recordings are converted to BIDS using [*phys2bids*](physio-to-bids.ipynb) and *etf2bids*, respectively.
    The logs from *Psychopy* are converted into BIDS' *events* files with the [*psychopy2events.py*](./post-session.md/#generate-bids-events-files) script.
    **Further processing steps, such as denoising, should be defined soon.**
```

Reply:
Oh sure - sorry about that !